src/components/views/read/PlayerView.tsx

```diff
@@ -86,24 +131,29 @@ const PlayerView: FC = () => {
         >
           <TextDisplay
             text={textResource}
-            keywords={keywords.keywords}
+            keywords={keywords}
             highlight={summon}
-            openChatbot={() => {
-              setChatbot(true);
+            openChatbot={(word: keyword) => {
+              openChatbot(word);
```

Comment:
Can you use this instead ?
```suggestion
            openChatbot={openChatbot}
```